src/appEventHandler.ts

```diff
@@ -1,14 +1,12 @@
-import { AppEvent } from '@grafana/data';
-import { SystemJS } from '@grafana/runtime';
+import { AppEvent, EventBusSrv } from '@grafana/data';
 import { getCalculationWarnings, getLimitsWarnings, stringifyError } from './cdf/client';
 import { failedResponseEvent, responseWarningEvent } from './constants';
 import { SuccessResponse } from './types';
 
-export const appEventsLoader = SystemJS.load('app/core/app_events');
+export const bus = new EventBusSrv();
```

Comment:
Renaming this to `busService` or `eventBusService` would` be a good idea, just `bus` feels a bit ambiguous